haiku-misc/vcardpeople/vcardpeople-1.0.1.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="Utilities to convert between VCard and BeOS 'Person' files"
+DESCRIPTION="The VCard format is an open standard used for storing and \
+interchanging personal (\"Business Card\") data by applications on many \
+platforms. The two small programs in this package will convert between this \
+format and BeOS 'Person' files, making it possible to share contact data \
+between BeOS and other platforms."
+HOMEPAGE="http://goodeveca.net/beos/"
+COPYRIGHT="2002 Peter J. Goodeve"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="http://goodeveca.net/beos/vcardpeople.zip"
+CHECKSUM_SHA256="ec46db14a069dbcec2d524b773b3d8043f056c77c1d7cccce8b6ce6504738f31"
+SOURCE_DIR="VCard-People"
+PATCHES="vcardpeople-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	vcardpeople = $portVersion
+	cmd:people2vcard = $portVersion
+	cmd:vcard2people = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:gcc
+	cmd:g++
```

Comment:
```diff
-	cmd:gcc
 	cmd:g++
+	cmd:gcc
 	cmd:ld
 	cmd:make
 	"
 
+defineDebugInfoPackage vcardpeople \
+	$binDir/people2vcard \
+	$binDir/vcard2people
+
+PATCH()
+{
+	sed -i \
+		-e "s|/boot/develop/etc/makefile-engine|`findpaths -r \
+		"makefile_engine" \
+		B_FIND_PATH_DEVELOP_DIRECTORY`/etc/makefile-engine|" \
+		SOURCE/*/Makefile
+}
+
 BUILD()
 {
+	export LD=g++
 	make -C SOURCE/PEOPLE_VCARD $jobArgs
 	make -C SOURCE/VCARD_PEOPLE $jobArgs
 }
```
It would be nice to upstream the fixes in order to avoid the `PATH()` function.
upstreaming the patch to avoid